pkg/gcv/configs/config.go

```diff
@@ -43,14 +44,14 @@ func init() {
 }
 
 const (
-	constraintGroup = "constraints.gatekeeper.sh"
+	ConstraintGroup = "constraints.gatekeeper.sh"
 	expectedTarget  = "validation.gcp.forsetisecurity.org"
-	yamlPath        = expectedTarget + "/yamlpath"
+	YamlPath        = expectedTarget + "/yamlpath"
```

Comment:
nit: dunno if we're following google style here... if we are, it should be YAMLPath per go/gocomments#initialisms

Reply:
lowercased it.